examples/distributed-solver/distributed-solver.cpp

```diff
@@ -79,6 +80,7 @@ int main(int argc, char* argv[])
         static_cast<gko::size_type>(argc >= 3 ? std::atoi(argv[2]) : 100);
     const auto num_iters =
         static_cast<gko::size_type>(argc >= 4 ? std::atoi(argv[3]) : 1000);
+    std::string schw_type = argc >= 5 ? argv[4] : "multi-level";
```

Comment:
Just as a note, I was running this example with
```
mpirun -n 4 ./distributed-solver reference 10000 1000
```
and the `one-level` version was faster and used less iteration. So maybe it should be the default.

Reply:
Yes, it seems to depend on the problem size. I see that 100000 multi-level reduces the number of iterations, but it is slower. But I will change the default to one-level.